FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiClientRepository.kt

```diff
@@ -159,16 +163,17 @@ object ApiClientRepository : BaseApiRepository {
             ApiHelper.configureOkHttpClientForDebuggingIfAllowed(context, this)
           }.build()
 
+        val genBaseFesUrlPath = GeneralUtil.genBaseFesUrlPath(true, domain)
```

Comment:
it would be good if the `true` had the `value=true` here for clarity. A boolean alone is not so clear (I know in IDE it's better, but still)